### PR TITLE
Fixes label length to be 128

### PR DESCRIPTION
**Pull request by sthirugn**
This fixes one of the jenkins failures.

Test results for this PR:
```sh
# nosetests tests/foreman/cli/test_product.py:TestProduct.test_positive_create_2
.
----------------------------------------------------------------------
Ran 1 test in 12.991s

OK

# nosetests tests/foreman/cli/test_product.py:TestProduct.test_positive_create_3
.
----------------------------------------------------------------------
Ran 1 test in 36.212s

OK
```

I also stubbed valid_labels_list and tested to check maximum value:
```py
def valid_labels_list():
    """List of valid labels for input testing."""
    return [
        gen_string('alphanumeric', 128),
        gen_string('alpha', 128),
    ]
```
Test result:
```sh
# nosetests tests/foreman/cli/test_product.py:TestProduct.test_positive_create_2
.
----------------------------------------------------------------------
Ran 1 test in 13.525s

OK
```

### CODE REVIEW

**Comment by elyezer** [comment]
ACK

File: robottelo/helpers.py [hunk 1]
```diff
@@ -246,8 +246,8 @@ def valid_data_list():
 def valid_labels_list():
     """List of valid labels for input testing."""
     return [
-        gen_string('alphanumeric', randint(1, 255)),
-        gen_string('alpha', randint(1, 255)),
+        gen_string('alphanumeric', randint(1, 128)),
+        gen_string('alpha', randint(1, 128)),
```
Comment (omaciel) [inline robottelo/helpers.py]: So you believe that labels cannot support strings longer than 128 chars?

File: tests/foreman/cli/test_product.py [hunk 2]
```diff
@@ -85,7 +85,7 @@ def test_positive_create_3(self):
         @Assert: Product is created and has random description
 
         """
-        for desc in valid_labels_list():
+        for desc in valid_data_list():
```
Comment (sthirugn) [inline tests/foreman/cli/test_product.py]: The description test should use valid_data_list and not valid_labels_list.  I found this when searching for the usage of valid_labels_list in robottelo.

File: robottelo/helpers.py (hunk 1 above)
Comment (sthirugn) [inline robottelo/helpers.py]: Yes I verified in satellite :)

**Comment by omaciel** [comment]
ACK

**Comment by sthirugn** [comment]
part of https://github.com/SatelliteQE/robottelo/issues/2769
